pages/getting-started/testing-your-app.md

```diff
@@ -0,0 +1,489 @@
+- Expor rotas específicas do ambiente de teste
+- Desativar recursos de segurança que tornam a automação difícil
+- Redefinir estado no servidor / banco de dados 
+
+Com isso dito - você ainda tem a opção de ter os **dois caminhos**. 
+
+Muitos de nossos usuários executam a maioria de seus testes de integração em um servidor de desenvolvimento local,
+mas reservam um conjunto menor de **testes de fumaça** que são executados apenas em uma aplicação de produção implantada.
+
+## Passo 2: Acesse seu servidor
+
+Assim que seu servidor estiver funcionando, é hora de acessá-lo.
+
+Vamos deletar o `sample_spec.js` arquivo criado no tutorial anterior que agora não é mais necessário. 
+
+```bash
+rm cypress/integration/sample_spec.js
+```
+
+Agora vamos criar nosso próprio arquivo de especificações chamado `home_page_spec.js`.
+
+```bash
+touch cypress/integration/home_page_spec.js
+```
+
+Assim que o arquivo for criado, você deve vê-lo na lista de arquivos de especificação.
+
+![home page spec](https://docs.cypress.io/img/guides/testing-your-app-home-page-spec.049e17f2.png)
+
+Agora você precisará adicionar o seguinte código em seu arquivo de teste para acessar seu servidor: 
+
+```javascript
+describe('The Home Page', () => {
+  it('successfully loads', () => {
+    cy.visit('http://localhost:8080') // change URL to match your dev URL
+  })
+})
+```
+
+Agora clique no arquivo `home_page_spec.js` e observe o Cypress abrir seu navegador.
+
+Se você se esqueceu de iniciar o servidor, verá o erro abaixo: 
+
+![visit fail](https://docs.cypress.io/img/guides/testing-your-app-visit-fail.10d4b941.png)
+
+Se você iniciou seu servidor, deverá ver sua aplicação carregada e funcionando. 
+
+## Passo 3: Configure Cypress
+
+[//]: <> (TODO - Adicionar link opção de configuração)
+Se você pensar no futuro, perceberá rapidamente que você digitará muito essa URL, 
+já que todo teste precisará visitar alguma página da sua aplicação. 
+Felizmente, o Cypress fornece uma 
+[opção de configuração](https://docs.cypress.io/guides/references/configuration.html) para isso. 
+Vamos aproveitar isso agora.
+
+Abra seu [arquivo de configuração](https://docs.cypress.io/guides/references/configuration.html) 
+(`cypress.json` no seu diretório do projeto, por padrão) ele começa vazio:
+
+```javascript
+{}
+```
+
+Vamos adicionar a opção `baseUrl`.
+
+```javascript
+{
+  "baseUrl": "http://localhost:8080"
+}
+```
+
+[//]: <> (TODO - Adicionar links dos comandos)
+Isso irá prefixar automaticamente os comandos [cy.visit()](https://docs.cypress.io/api/commands/visit.html) 
+e [cy.request()](https://docs.cypress.io/api/commands/request.html) com este baseUrl. 
+
+> Sempre que você modificar seu arquivo de configuração, o Cypress irá reiniciar-se automaticamente e 
+encerrar todos os navegadores abertos. Isto é normal. Clique no arquivo de especificações novamente para 
+reiniciar o navegador.
+
+Nós agora podemos o caminha relativo e omitir o nome do host e a porta.
```

Comment:
Essa frase parece errada. Além disso, é caminho, não caminha.

Reply:
@wlsf82 a frase é essa aqui: "We can now visit a relative path and omit the hostname and port." e foi traduzida agora para "Nós agora podemos visitar o caminho relativo, e omitir o nome do host e a porta."